baselines/jft/experiments/vit_l32_active_learning.py

```diff
@@ -43,6 +43,7 @@ def get_config():
   config.test_split = ''  # set in sweep
   config.val_split = ''  # set in sweep
   config.train_split = ''  # set in sweep
+  config.test_split = 'test'
```

Comment:
@zi-w this line would be good to double check. I know there's been some discussion around the "right" test split.

Reply:
Ah okay, then test_split can be removed here again